src/bindings/README.md

```diff
@@ -19,6 +19,7 @@ List of currently supported bindings (use `ALL;-EXPERIMENTAL;-DEPRECATED`):
 - [swig_python2](swig/python2/) Python 2 SWIG bindings
 - [swig_ruby](swig/ruby/) Ruby bindings
 - [jna](jna/) A full java binding using JNA
+- [rust](rust/) Bindings for the low-level API in Rust
```

Comment:
The term "low-level" is only used here. (I think the whole section needs to be rewritten, I can also do it.)